onnxruntime/core/providers/dnnl/subgraph/dnnl_conv.h

```diff
@@ -468,34 +465,34 @@ class DnnlConv : public DnnlKernel {
   }
 
  private:
-  void ReadAttributes(const NodeAttributes& attributes,
+  void ReadAttributes(const Prov_NodeAttributes& attributes,
                       const std::string attributes_prefix = "") override {
     std::string auto_pad;
     auto attr = attributes.find(attributes_prefix + "auto_pad");
     if (attr != attributes.end() &&
-        attr->second.type() == ::ONNX_NAMESPACE::AttributeProto_AttributeType::AttributeProto_AttributeType_STRING) {
-      auto_pad = attr->second.s();
+        attr->second->type() == ::ONNX_NAMESPACE::AttributeProto_AttributeType::AttributeProto_AttributeType_STRING) {
+      auto_pad = attr->second->s();
     }
     auto_pad_ = (auto_pad != "") ? StringToAutoPadType(auto_pad) : AutoPadType::NOTSET;
 
     kernel_shape_specified_ = false;
     attr = attributes.find(attributes_prefix + "kernel_shape");
     if (attr != attributes.end()) {
-      ONNX_NAMESPACE::AttributeProto proto = attr->second;
+      auto& proto = *attr->second;
```

Comment:
const auto& and other places too.

Reply:
Hmm, I get a build error if I try to make this const. It wasn't const before, is there something I'm missing?